app/src/main/java/org/mozilla/fenix/search/awesomebar/AwesomeBarView.kt

```diff
@@ -207,6 +212,18 @@ class AwesomeBarView(
         searchSuggestionProviderMap = HashMap()
     }
 
+    private fun getDefaultSearchEngine(components: Components): String? {
+        var defaultSearchEngine = components.core.store.state.search.selectedOrDefaultSearchEngine?.name
+
+        if (!defaultSearchEngine.isNullOrEmpty()) {
+            defaultSearchEngine = when (defaultSearchEngine) {
+                HomeFragment.GOOGLE_SEARCH_ENGINE_NAME -> "$defaultSearchEngine Search"
+                else -> "$defaultSearchEngine search"
```

Comment:
I would imagine we need to localized these strings here. You can have strings that can take a parameter (eg, defaultSearchEngine). See https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/res/values/strings.xml#435 and https://searchfox.org/mozilla-mobile/source/fenix/app/src/main/java/org/mozilla/fenix/settings/HomeSettingsFragment.kt#106.

When you define the string takes a parameter, you also need to specify what the parameter is in localization comment.